### DISCUSSION TITLE
Windows Taskmanager multiple instances of Electron

### DISCUSSION THREAD

**Discussion by osc2nuke** [NativePHP/laravel#172] [Q&A] [answered]
Yesterday when i took a break from coding and all programs closed, my machine kept running.
When i checked windows taskmanager i seen 4 tasks of Electron running in the background eating resources.
I wonder if someone knows how/why?
I not made a nativephp:build yet (not that far lol). Only used nativephp:serve.

**Comment by osc2nuke** [comment]
Not yet sure, but happens when i run the app with high tasks.
In my case a 35MB what needs to be read, and inserted to SQLITEdb.
Now i use smaller files for testing/dev.
Everyting quiet now.

**Comment by simonhamp** [comment, marked as answer]
A lot of updates have been made to the process management side of NativePHP in recent months, so this should be less on of an issue. Please feel free to drop back in here if you experience further stray processes.
